app/helpers/comments_helper.rb

```diff
@@ -0,0 +1,2 @@
+module CommentsHelper
```

Comment:
Is this to help with the routing for displaying comments? Just trying to make sure I keep up with the code.

Reply:
No, this is autogenerated with rails g controller, not sure if we really need it

Reply:
Ok sounds good, we'll probably figure out if it's helpful in testing